### PR TITLE
Work in progress

### PR DESCRIPTION
**Pull request by Israel-Laguan**
# Resume

Mario and I reviewed the form and compare it to the original and came to the conclusion that we finished the styling correctly. 

# Proposal

We want to have a review of our code to know if we need to make any changes. If no changes needed we want the project to be ACCEPTED so we can move to the next project with a happy smile :happy:

# Comments

in comparison to Youtube-Clone project it was easier but we wanted to take the oportinity to improve our workflow. Thats why we have separate branches for the layout elements. Also, we tried to make sure to have a good style close to the original.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+                  id="email"
+                />
+              </div>
+            </div>
+            <div class="phone">
+              <label for="phone" class="formLabel">
+                Phone (Recommended)
+              </label>
+              <div class="input">
+                <input class="input__inputBar" type="text" id="phone" />
```
Comment (Ispirett) [inline index.html]: The type here should be `type='tel'`

Comment (MarioDena) [reply index.html]: Thank you!

Comment (MarioDena) [reply index.html]: By the way can you gine us your name we need it to submit the project competition

**Comment by Israel-Laguan** [comment]
Thanks man!! Really apreciated!

**Comment by MarioDena** [comment]
@Ispirett Can you please give us your name we need it to submit the project. =)

**Comment by Israel-Laguan** [comment]
> 
> 
> @Ispirett Can you please give us your name we need it to submit the project. =)

Please mr @Ispirett

**Pull request merged**